Makefile

```diff
@@ -4,4 +4,7 @@ clean:
 
 .PHONY: profile
 profile:
-	zip -r -9 profile/webapps/system.gaiamobile.org/application.zip assets bower_components css index.html js manifest.webapp
+	zip -r -9 profile/webapps/system.gaiamobile.org/application.zip assets index.html lib node_modules js manifest.webapp
+
+addon:
+	zip addon.zip addon.js manifest.json assets index.html lib node_modules
```

Comment:
I'm confused why we need to manually zip this thing up. Doesn't WebIDE do this work for us?